adapters-api/src/main/java/io/nosqlbench/engine/api/activityconfig/rawyaml/OpsOwner.java

```diff
@@ -20,21 +20,21 @@
 
 import java.util.*;
 
-public class StatementsOwner extends RawStmtFields {
+public class OpsOwner extends RawOpFields {
 
     private final static List<String> stmtsFieldNames = List.of("op","ops","operation","statement","statements");
 
-    private List<RawStmtDef> rawStmtDefs = new ArrayList<>();
+    private List<RawOpDef> rawOpDefs = new ArrayList<>();
 
-    public StatementsOwner() {
+    public OpsOwner() {
     }
 
-    public List<RawStmtDef> getRawStmtDefs() {
-        return rawStmtDefs;
+    public List<RawOpDef> getRawStmtDefs() {
```

Comment:
Do you still want this to remain Stmt or Op for the defs given it returns RawOpDef?

Reply:
I missed this one, I'll update it.

Reply:
making a separate PR which improves naming of everything